### PR TITLE
Updated Contributing.md

### PR DESCRIPTION
**Pull request by maple0311**
Screenshot (43)
@JLambertazzo please look into it and suggest further changes if required.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,9 +4,97 @@
...
+
+# Project Outline 
+
+## For core library and logic 
+- Checkout [lib](/lib/) folder.
+- Also , for more instructions on using the library, visit:
+https://julienbl.me/ListExtender/documentation.html &nbsp; &nbsp; &nbsp; OR run [documentation](/documentation.html) on localhost.
+
+## Checkout the website
+- Click [html/website](/index.html) and run on localhost.
```
Comment (JLambertazzo) [inline CONTRIBUTING.md:89]: Could you update "html/website" to be something a bit more explicitly referencing index.html? Just the way the format is at the moment is a bit confusing where to look

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -4,9 +4,97 @@
...
+## Step 9: Merged?
+
+You will receive a notification email once your changes have been merged.
+
+- For more help on the process visit [Hacktoberfest](https://hacktoberfest.digitalocean.com/resources) !
+
+# Project Outline 
+
+## For core library and logic 
+- Checkout [lib](/lib/) folder.
```
Comment (JLambertazzo) [inline CONTRIBUTING.md:84]:
```suggestion
+ Checkout [lib](https://github.com/JLambertazzo/ListExtender/tree/master/lib) folder.
```
Could you update the absolute paths (`/something`) to links using the full url like above? I'm concerned that after merge the absolute paths will not lead to the correct place

**Review by JLambertazzo — COMMENTED** [review]
A couple of small notes. Thank you for this huge overhaul to the contributing guidelines!! Your explanation of the PR process shows a great understanding of open source collaboration, I'm sure this hacktoberfest will be a hugely successful one for you 🚀🎃

**Comment by jks1111** [comment]
Thank you . will fix it.

**Comment by jks1111** [comment]
@JLambertazzo , Sorry for the delay. Please look into the changes.

**Comment by JLambertazzo** [comment]
@jks1111 Sorry I didn't get to this last year! Let me know if/when you've registered for this year's hacktoberfest and we'll get this merged.

**Review by JLambertazzo — APPROVED** [review]
Looks great!! Happy hacktoberfest 🎃 🚀

**Pull request merged**